src/main/java/uk/modl/parser/ModlParsed.java

```diff
@@ -331,23 +418,35 @@ public ValueConditional getValueConditional() {
         ValueItem valueItem;
 
         @Override
-        public void enterPair(MODLParser.PairContext ctx) {
+        public void enterModl_pair(MODLParser.Modl_pairContext ctx) {
             if (ctx.STRING() != null) {
-                key = ctx.STRING().toString();
+                key =
+                        ctx
+                                .STRING()
+                                .toString();
             }
             if (ctx.QUOTED() != null) {
-                key = ctx.QUOTED().toString();
+                key =
+                        ctx
+                                .QUOTED()
+                                .toString();
```

Comment:
Do we not need the additionalStringProcessing here as well?

Reply:
Possibly not needed here because its extracting the key part of the pair rather than the value, and it doesn't look like object references are supported for keys - at least it doesn't seem to be mentioned explicitly.